src/parser/AdminSentences.cpp

```diff
@@ -24,6 +24,46 @@ std::string ShowPartsSentence::toString() const {
     return std::string("SHOW PARTS");
 }
 
+std::string ShowTagsSentence::toString() const {
```

Comment:
Schema is belonged to maintain

Reply:
Actually I want to move `show sentence` into one place.

Reply:
The classification is consistent with the validate and executor. I don’t think it needs to move.

Reply:
Emmm ... The core of this problem is that what do you think about `show` sentence. 

IF you think it is a admin sentence, you should put them together. 

According to you, `ShowUsersSentence` should put into `UserSentence`

Reply:
Actual the `ShowUsersSentence` also belongs to the `admin`. Everything related to the user belongs to `Admin`.